pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
I think it was fixed in 1.5.4. @colemickens would know better than me. I just know the const we save which she was mentioning should use 1.5.1 for our 1.5.3 cluster.